### DISCUSSION TITLE
dom question

### DISCUSSION THREAD

**Discussion by akramkhan8998** [programmingashram/ashram-teams#227] [Q&A]
how we access a div in dom for different Semantic Elements 
let section = document.createElement('section');
body.appendChild(section);
section.appendChild(div);
div.setAttribute('id','div-2');
let div-2 = document.getElementById('div-2');
is **_`[this a correct way to access means how be access another  way to do ,this is not working right now

**Comment by nimisha666** [comment]
let body = document.getElementById('body');

let section = document.createElement('section');
body.appendChild(section);

let div = document.createElement('div');
section.appendChild(div);
div.setAttribute('id','div2');

div.innerHTML = `<p>nisha</p>`

**Reply by akramkhan8998** [reply]
This is not correct way at all because div. Is also declare earlier
